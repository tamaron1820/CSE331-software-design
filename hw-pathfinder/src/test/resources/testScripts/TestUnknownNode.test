# Test if unknown nodes are called

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2

FindPath graph1 n1 n9

FindPath graph1 n2 n5

FindPath graph1 n3 n4