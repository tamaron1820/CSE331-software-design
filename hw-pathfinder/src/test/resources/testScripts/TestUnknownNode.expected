# Test if unknown nodes are called

created graph graph1

added node n1 to graph1
added node n2 to graph1

unknown: n9

unknown: n5

unknown: n3
unknown: n4